libcxx/include/__algorithm/radix_sort.h

```diff
@@ -0,0 +1,415 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
```

Comment:
Not attached: Please add a release note about the improvement.

Reply:
Can you please give an example of how to do it?

Reply:
Simply something like "`std::stable_sort` uses radix sort for integral types now, which can improve the performance by <the improvement>".

Reply:
Done.